oaknational/Oak-Web-Application
# Fix/cur 417/incorrect ordering of units in visualiser

## Description

Curriculum units are being displayed in the wrong order - See the ticket below.

https://www.notion.so/oaknationalacademy/Incorrect-ordering-of-units-in-visualiser-e4dd201f74f4424ebfec7be2296b82d6

## Fix

- The order field is now brought through with the MV and used within the getStaticProps to order the units correctly.

## How to test

1. Go to https://deploy-preview-2320--oak-web-application.netlify.thenational.academy/teachers/curriculum/maths-secondary/units
2. Click on Year 10
3. You should see units in the correct order. Please consult the CAT for correct ordering.

1. Go to https://deploy-preview-2320--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-primary/units
2. Year 1 - 'Ada Twist Scientist' should be in position 17.
3. Year 2 - 'Monster Pizza' should be in position 14.
3. Year 2 - 'Apostrophes' should be in position 3.

## Screenshots
Not required

## Checklist
